tests/MailboxTest.php

```diff
@@ -101,6 +101,8 @@ public function testGetMessageSequence(): void
     public function testGetMessageSequenceThrowsException(): void
     {
         $this->expectException(InvalidSearchCriteriaException::class);
+        $this->expectExceptionMessageRegExp('/Syntax error in sequence/');
```

Comment:
This test succeeds even without your changes in Mailbox, and this is because all the errors are outputted in the exception trace:
```
Ddeboer\Imap\Exception\InvalidSearchCriteriaException: Invalid sequence [-1:x]
imap_alerts (0):
imap_errors (1):
- Syntax error in sequence
```
See https://github.com/ddeboer/imap/blob/master/src/Exception/AbstractException.php#L44

So as far as I can tell, there is no need for this PR :\